<launch>
 <!-- Industrial shields-->
  <!--<node pkg="rosserial_python" type="serial_node.py" name="Industrial_sheild" output="screen">
    <param name="port" value="/dev/aragorn_indus"/>
    <param name="baud" value="115200"/>
    
  </node>-->

    <node pkg="rosserial_python" type="serial_node.py" name="imu_Teensy" output="screen">
      <param name="port" value="/dev/ttyACM0"/>
      <param name="baud" value="115200"/>
    </node>

    <node pkg="imu_calib" type="apply_calib" name="apply_calib" output="screen" respawn="false">
        <param name="calib_file" value="$(find ros_zlac706)/param/imu/imu_calib.yaml" />
        <param name="calibrate_gyros" value="true" />
    </node>

  <!-- Filter and fuse raw imu data -->
    <node pkg="imu_filter_madgwick" type="imu_filter_node" name="imu_filter_madgwick" output="screen" respawn="false" >
        <param name="fixed_frame" value="base_footprint" />
        <param name="use_mag" value="true" />
        <param name="publish_tf" value="false" />
        <param name="use_magnetic_field_msg" value="true" /> 
        <param name="world_frame" value="enu" />
        <param name="orientation_stddev" value="0.05" />
    </node>

    
</launch>
